include/os/linux/kernel/linux/simd_powerpc.h

```diff
@@ -80,40 +80,32 @@
 #define	ENABLE_KERNEL_VSX
 #define	DISABLE_KERNEL_VSX
 #endif
-#ifdef CONFIG_SPE
-#define	kfpu_begin()				\
-	{					\
-		preempt_disable();		\
-		ENABLE_KERNEL_ALTIVEC		\
-		ENABLE_KERNEL_VSX		\
-		enable_kernel_spe();		\
-	}
-#define	kfpu_end()				\
-	{					\
-		disable_kernel_spe();		\
-		DISABLE_KERNEL_VSX		\
-		DISABLE_KERNEL_ALTIVEC		\
-		preempt_enable();		\
-	}
-#else /* CONFIG_SPE */
+#ifdef	CONFIG_SPE
+#define	ENABLE_KERNEL_SPE	enable_kernel_spe()
+#define	DISABLE_KERNEL_SPE	disable_kernel_spe()
```

Comment:
From:
```
#define	ENABLE_KERNEL_SPE	enable_kernel_spe()
#define	DISABLE_KERNEL_SPE	disable_kernel_spe()
```

To:
```
#define	ENABLE_KERNEL_SPE	enable_kernel_spe();
#define	DISABLE_KERNEL_SPE	disable_kernel_spe();
```

Then I will approve ;)

Reply:
Why the preference for including the `;` in the #define?  This seems like it should be purely cosmetic, and our usual convention is to not hide it in the macro.  Though I don't feel strongly about it.

Reply:
Yes, it's just cosmetic. My variant just defines it `all away` - which seems nicer to me.
But when there is a usual convention which does does not hide the `;` - then we should follow this convention.

@Low-power - sorry for the noise ;-)